BoxContentSDK/BoxContentSDK/Models/BOXFile.h

```diff
@@ -91,4 +91,11 @@
 
 @property (nonatomic, readwrite, strong) NSArray *representations;
 
+
+/**
+ *  Url available to download original content.
+ *  Warning: file exists only if canDownload is true
+ */
+@property (nonatomic, readwrite, strong) NSURL *downloadUrl;
```

Comment:
Not specific to this PR but throw in a nullable attribute here?

Reply:
Agree, chatted offline.  A good discussion for the team, to go back into ContentSDK and add Nullability attributes.